samples/core/guide/autograph.ipynb

```diff
@@ -609,6 +646,217 @@
       "execution_count": 0,
       "outputs": []
     },
+    {
+      "metadata": {
+        "id": "XY4UspHmZNdL",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "## Interoperation with `tf.Keras`\n",
+        "\n",
+        "Now that you've seen the basics, let's build some real model components with autograph.\n",
+        "\n",
+        "It's relatively simple to integrate `autograph` with `tf.keras`. But remember that batchng is essential for performance. So the best candidate code for conversion to autograph is code where the control flow is decided at the _batch_ level. If decisions are made at the individual _example_ level you will still need to index and batch your examples to maintain performance while appling the control flow logic. \n",
+        "\n",
+        "\n",
+        "### Stateless functions\n",
+        "\n",
+        "For stateless functions like `collatz`, below, the easiest way to include them in a keras model is to wrap them up as a layer uisng `tf.keras.layers.Lambda`."
```

Comment:
For stateless functions, like `collatz` shown below, the easiest way to include them in a Keras model is to wrap them up as a layer using `tf.keras.layers.Lambda`:

Reply:
Done